core/templates/pages/topic-viewer-page/subtopics-list/subtopics-list.directive.html

```diff
@@ -18,9 +18,9 @@
        ng-swipe-left="scroll(false)">
     <div class="oppia-topic-viewer-carousel-tiles">
       <md-card ng-repeat="subtopic in subtopics" class="oppia-subtopic-summary-tile">
-        <subtopic-summary-tile skill-count="subtopic.skill_ids.length"
-                               subtopic-id="subtopic.id"
-                               subtopic-title="subtopic.title"
+        <subtopic-summary-tile skill-count="subtopic.getSkillSummaries().length"
+                               subtopic-id="subtopic.getId()"
```

Comment:
Ditto as above*

Reply:
![image](https://user-images.githubusercontent.com/32016324/79643822-71bbe800-81c2-11ea-977b-c93b1362774a.png)

Reply:
Thanks for checking! :)